.github/workflows/build-test-measure.yml

```diff
@@ -216,7 +216,7 @@ jobs:
         id: get-comment-body
         # Setting a multi-line output requires escaping line-feeds. See <https://github.community/t/set-output-truncates-multiline-strings/16852/3>.
         run: |
-          body="Plugin builds for ${{ github.sha }} are ready :bellhop_bell:!
+          body="Plugin builds for ${{ github.event.pull_request.head.sha }} are ready :bellhop_bell:!
```

Comment:
Oh, hold on... What about for branch builds?

Reply:
Not sure what you mean, this wouldn't occur for branch builds.

Reply:
It wouldn't? What about PRs merged into `develop` or commits cherry-picked onto the `1.5` branch?

![image](https://user-images.githubusercontent.com/134745/84549641-57335600-acbd-11ea-9fbf-17db4ab735e6.png)

Reply:
I meant the job that performs the commenting will only run for PR builds. Otherwise the job is skipped, as shown in the screenshot above.

Reply:
Oh of course. 🤦